### DISCUSSION TITLE
JavaScript Promise va Async/Await

### DISCUSSION THREAD

**Discussion by giscus** [IkboljonMe/blog#2] [Ideas]
# JavaScript Promise va Async/Await

JavaScriptdagi Promise va qanday qilib 'Callback Hell' muammosidan qutilish

http://localhost:3002/blog/javascript-promiselar

**Comment by IkboljonMe** [comment]
This is test comment

**Comment by mrspecht** [comment]
Async/await all the way. It looks more 'natural' since you have a function to handle an asynchronous task
